frontend/tailwind.config.ts

```diff
@@ -82,6 +82,9 @@ export default <Partial<Config>>{
           instagram: "rgba(129, 52, 175, 1)",
         },
       },
+      transitionProperty: {
```

Comment:
Just for a bit of clarity, I added this here as I noticed that `transition-width` was being used but it is not included by default. It now works as expected. 
Example: [PasswordStrength](https://github.com/activist-org/activist/blob/main/frontend/components/PasswordStrengthIndicator.vue#L7)

Reply:
Really great to have the attention to these sorts of things, @lucianmurmurache :) Thank you!